From 5c0703f06201793d71f80735ae61c6cdcdeaf954 Mon Sep 17 00:00:00 2001
From: zhangql0721 <zhangql0721@thundersoft.com>
Date: Thu, 17 Sep 2020 11:36:58 +0800
Subject: [PATCH 3/3] testpach

---
 hello.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hello.c b/hello.c
index b5cbccb..b9a2f5a 100644
--- a/hello.c
+++ b/hello.c
@@ -5,5 +5,6 @@ void main(void){
 	/*real want to modify */
 	/* want to modfiy test mergen */
 	/* test mergetn in repo test */
+	printf("test patch\n");
 
 }
-- 
2.17.1

